spec/javascripts/components/SlicePanels.test.js

```diff
@@ -47,10 +47,35 @@ const helpers = {
 };
 
 SpecSugar.withTestEl('high-level integration tests', (container) => {
-  it('renders everything on the happy path', () => {
+  it(`renders everything on the happy path for elementary school with
+      no student registration dates`, () => {
     const el = container.testEl;
     helpers.renderInto(el);
 
+    expect($(el).find('.SlicePanels').length).toEqual(1);
+    expect($(el).find('.column').length).toEqual(6);
+    expect(helpers.columnTitlesMatrix(el)).toEqual([
+      [ 'Disability', 'Low Income', 'LEP', 'Race', 'Hispanic/Latino', 'Gender' ],
+      [ 'Grade', 'Risk level' ],
+      [ 'STAR Reading', 'MCAS ELA Score', 'MCAS ELA SGP' ],
+      [ 'STAR Math', 'MCAS Math Score', 'MCAS Math SGP' ],
+      [ 'Discipline incidents', 'Absences', 'Tardies' ],
+      [ 'Services', 'Summer', 'Notes', 'Program', 'Homeroom' ]
+    ]);
+  });
+
+  it(`renders everything on the happy path for elementary school with
```

Comment:
If this is about differences in data systems across schools, let's add that to the comment.